OUDS/Core/OUDS/Sources/OUDSTheme/OUDSTheme+SemanticTokens/OUDSTheme+DimensionSemanticTokens.swift

```diff
@@ -19,6 +19,6 @@ import OUDSTokensSemantic
 /// These values can be overriden inside `OUDSTheme` subclasses (in extensions or not, in the same module or not) thanks to the `@objc open` combination.
 extension OUDSTheme: DimensionSemanticTokens {
 
-    @objc open var dimensionDensityDefault: DimensionSemanticToken { DimensionRawTokens.dimensionBase }
+    @objc open var dimensionDensityDefault: DimensionSemanticToken { 4 }
```

Comment:
why is it hard coded ?  A sementic token should (must) be defined by a raw token, right ?
So here should be DimensionRawTokens.dimension200.

Reply:
You are right. It seems some tokens have been added, "sys" versions of semantic tokens, I will update.